### PR TITLE
Add channel and componentsOnce to README

### PR DESCRIPTION
**Pull request by amberroy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,15 +22,22 @@ If you want to allow unauthenticated users (most should), then go into
 
 #### firebase
 
-Firebase configuration component for `<a-scene>`. Firebase will provide all of
-this for you when you set up a Firebase app.
+Firebase configuration component for `<a-scene>`. Firebase will provide the 
+first four properties when you set up a Firebase app. You select the optional
```
Comment (ngokevin) [inline README.md]: maybe specify `apiKey, authDomain, databaseUrl, and storageBucket` rather than first four

Comment (ngokevin) [inline README.md]: i'd put channel info on separate paragraph and `s/you/we`

File: README.md [hunk 2]
```diff
@@ -22,15 +22,22 @@ If you want to allow unauthenticated users (most should), then go into
...
+channel name (if none given, 'default' is used).  Channels allow multiple apps,
+or multiple instances/rooms of the same app, to share one Firebase bucket.  
 
 | Property      | Description                     |
 | --------      | -----------                     |
 | apiKey        | API key for Firebase.           |
 | authDomain    | Firebase authentication domain. |
 | databaseURL   | Firebase database URL.          |
 | storageBucket | Firebase storage bucket URL.    |
+| channel       | Name of room/namespace          |
```
Comment (ngokevin) [inline README.md]: would put a white line after this, and period the desc, and alphabetize the props

File: README.md [hunk 3]
```diff
@@ -22,15 +22,22 @@ If you want to allow unauthenticated users (most should), then go into
...
+The channel name can also be specified in the URL:
```
Comment (ngokevin) [inline README.md]: would put this in the same paragraph as the channel info above

File: README.md [hunk 4]
```diff
@@ -39,6 +46,11 @@ Broadcast component data to be synced across all clients using Firebase realtime
 | Property   | Description                                          | Default Value      |
 | --------   | -----------                                          | -------------      |
 | components | List of comma-delimited component names to broadcast | position, rotation |
+| componentsOnce | Sync initial value only; for components that don't change | 
+
+For example: `firebase-broadcast="components: material, geometry"`
```
Comment (ngokevin) [inline README.md]: Can use html syntax highlighting

For example:

```html
firebase-broadcast="components: material, geometry"
```

To broadcast...

File: README.md (hunk 1 above)
Comment (amberroy) [inline README.md]: done.

File: README.md (hunk 2 above)
Comment (amberroy) [inline README.md]: done.

File: README.md (hunk 3 above)
Comment (amberroy) [inline README.md]: done.

File: README.md (hunk 4 above)
Comment (amberroy) [inline README.md]: done.

**Comment by ngokevin** [comment]
Thanks!

**Pull request merged**